packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -88,15 +103,15 @@ const useStyles = makeStyles({
 
   sizeSmall: {
     [cardCSSVars.cardSizeVar]: '8px',
-    ...shorthands.borderRadius(tokens.borderRadiusSmall),
+    [cardCSSVars.cardBorderRadiusVar]: tokens.borderRadiusSmall,
```

Comment:
Creating a var felt like the better option, given that otherwise we'd need the `createFocusOutlineStyle` call three times, one for each size, as the radius changes.
It also wouldn't be directly tied to the actual border radius, so changing in one place and not the other would bring inconsistencies.

Reply:
yes this pattern is great and just showcases the power of css variables although creates a kind of "dependency injection confusion :D"